Add: local sistent package usage to Sistent About Page

M-DEV-1 opened the pull request:
**Description**

This PR fixes #6402 

**Notes for Reviewers**
[Screencast from 2025-05-15 17-37-04.webm](https://github.com/user-attachments/assets/71af5e54-44d8-4bcc-88eb-dc1c510c7fe5)



**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

<!--
Thank you for contributing to Layer5 projects! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

M-DEV-1 commented:
[Screencast from 2025-05-15 17-37-04.webm](https://github.com/user-attachments/assets/9aa0b645-6a64-4bde-8b0f-7e11bb6024f5)

leecalcote commented on src/sections/Projects/Sistent/about.js:
```diff
@@ -101,9 +102,62 @@ const SistentAbout = () => {
+            </p>
+
+            <p>
+              To contribute to Layer5 projects' UI using Sistent e.g.{" "}
```
```suggestion
              To contribute to projects using Sistent e.g.{" "}
```

leecalcote commented on src/sections/Projects/Sistent/about.js:
```diff
@@ -101,9 +102,62 @@ const SistentAbout = () => {
-              To contribute to Layer5 projects' UI using Sistent e.g. <a href="https://github.com/meshery/meshery/issues?q=is%3Aissue%20state%3Aopen%20label%3Acomponent%2Fui">meshery-ui</a> and others, You can
-              refer to {" "}
+              This will remove the local Sistent package from your project. You
+              will have to install the standard package again after running this
```
List the command.

leecalcote commented on src/sections/Projects/Sistent/about.js:
```diff
@@ -101,9 +102,62 @@ const SistentAbout = () => {
+              </li>
+            </ol>
+            <p>
+              Now your project should reflect changes from your local Sistent
```
```suggestion
              Now, your project should reflect changes from your local Sistent
```

leecalcote commented on src/sections/Projects/Sistent/about.js:
```diff
@@ -115,7 +169,7 @@ const SistentAbout = () => {
                 Websites call
               </a>{" "}
-              where experienced contributors have taught how to use sistent in
+              where experienced contributors have taught how to use Sistent in
```
Incorrect English phrase.

vishalvivekm replied:
accepting suggestions // @M-DEV-1

leecalcote reviewed (changes requested):
No `link`?

vishalvivekm commented:
@M-DEV-1 
Thank you for your contribution!
Let's discuss this during the website call today at 7 AM CT | 5:30 PM IST.

Add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk), if you would :)

winkletinkle commented:
@vishalvivekm @M-DEV-1 was this discussed? What is the outcome? Why is there no clear path forward here?

winkletinkle commented:
@leecalcote, I can update the text and get this job done, if needed.

leecalcote commented:
@winkletinkle, thank you. Hold that thought. I'm hopeful that either one of these other two contributors will finish the job that was started about a week without me having to follow up with them again.

vishalvivekm commented on src/sections/Projects/Sistent/about.js:
```diff
@@ -101,9 +102,62 @@ const SistentAbout = () => {
+              This will remove the local Sistent package from your project. You
+              will have to install the standard package again after running this
+              command.
+            </p>
```
```suggestion
            </p>
            <Code codeString="npm install @layer5/sistent" />
```

vishalvivekm commented on src/sections/Projects/Sistent/about.js:
```diff
@@ -101,9 +102,62 @@ const SistentAbout = () => {
-              refer to {" "}
+              This will remove the local Sistent package from your project. You
+              will have to install the standard package again after running this
+              command.
```
```suggestion
              command:
```

l5io commented:
🚀 Preview for commit a0c4048e15e706527b01852e1dcd6edafc3f4324 at: https://682da94b1cdba34d438aed6b--layer5.netlify.app

vishalvivekm commented:
> @vishalvivekm @M-DEV-1 was this discussed? What is the outcome? Why is there no clear path forward here?

Hi @winkletinkle, yes this was discussed and @M-DEV-1 was to make accept suggestions and make any remaining updates.

M-DEV-1 commented:
@vishalvivekm thank you for fixing the grammar, I've added instructions for `npm link`. I've added the `do-not-merge` label because I'm doing a few tests.

l5io commented:
🚀 Preview for commit 0770362e52cf37a78a19b4243a6a64eaaf897136 at: https://682dac71300da6b056229f91--layer5.netlify.app

l5io commented:
🚀 Preview for commit 8917182f1ce107252b78493dd9f68147ce0a2e8f at: https://682dbcfa72cfb874d5f151e2--layer5.netlify.app

M-DEV-1 commented:
I've also updated the docs in the Sistent repo with `npm link`

l5io commented:
🚀 Preview for commit 0988ca4e86a302d360c4a690f3aef45de6e51b18 at: https://6836e41ad77cc94680e851b5--layer5.netlify.app

vr-varad commented:
![image](https://github.com/user-attachments/assets/d939659b-fd45-455c-8b72-2cf48b5cd023)
![Screenshot from 2025-05-29 20-10-32](https://github.com/user-attachments/assets/6f931c62-fb5b-4076-b627-1f487a60333a)


hey @M-DEV-1 mind remove the numbers (list)
1. Method 1

doesn't make sense

l5io commented:
🚀 Preview for commit 24c4a4b77f6d06d97f1f63db0162c7078d4b37de at: https://68398c8378f84c4c3d781048--layer5.netlify.app

M-DEV-1 commented:
thanks @vr-varad, it's fixed now.

https://68398c8378f84c4c3d781048--layer5.netlify.app/projects/sistent/about

l5io commented:
🚀 Preview for commit 5a9121fe043cad0d2d54586cb42f4a208b2c2171 at: https://685e5bdfc651b3ae51948196--layer5.netlify.app

vishalvivekm reviewed (changes requested):
@M-DEV-1 
Please remove from the nested lists.
They aren't offering great viewing experience: 
![Screenshot_20250628-145437](https://github.com/user-attachments/assets/79fcece2-7703-4c3f-9fee-88748bf08e29)

l5io commented:
🚀 Preview for commit e0c92bfe9c33d0892cb0c3787cfe021fc4c587ca at: https://685fbcfe8482fc39a13147a5--layer5.netlify.app

M-DEV-1 commented:
@vr-varad this PR is complete

l5io commented:
🚀 Preview for commit 639990b786fe9355399673d54c5e7b09704fdab3 at: https://68991b682071a57ec5683e2d--layer5.netlify.app

l5io commented:
🚀 Preview for commit d7e1b92a9c72c3ecf84f8774dc1a95778a0c1888 at: https://68b064cd5ba078575da6e555--layer5.netlify.app
